packaging/windows/clisdk/generatebundle.ps1

```diff
@@ -113,6 +113,4 @@ if(!(Test-Path $DotnetBundleOutput))
 
 Write-Host -ForegroundColor Green "Successfully created dotnet bundle - $DotnetBundleOutput"
 
-_ $RepoRoot\test\Installer\testmsi.ps1 @("$CLISDKMSIFile")
```

Comment:
It's still happening, check msi targeets.

I don't know why this call was here. It's a hack